## 82% of end-end tests have passed
```
Total executed tests: 11
Failed tests: 2
Expected to fail tests: 0
Passed tests: 9
```
```
IDs of failed tests: 702742,703133 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/keycard-theme&PR_ID=22028&APK_NAME=22028.apk&TR_CASE_IDS=702742,703133">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781089">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><p><blockquote><div>Device 1: Double tap on: Button </div><div>Device 1: Find `Button` by `xpath`: `//*[@text="Copy text"]` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:60: in test_community_copy_and_paste_message_in_chat_input
    self.channel.copy_message_text(message)
../views/chat_view.py:673: in copy_message_text
    self.element_by_translation_id("copy-text").double_click()
../views/base_element.py:113: in double_click
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:113: in <listcomp>
    [self.find_element().click() for _ in range(2)]
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@text="Copy text"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6Y1E7DObvdA3T0d3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781092">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Check that removed user is not shown in the list anymore</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:246: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: zQ3...dWXh5 was not restored as a contact from waku backup!
E    Device 1: zQ3...Vacac was not restored as a contact from waku backup!
E    Device 1: admin_open was not restored from waku-backup!!
E    Device 1: member_open was not restored from waku-backup!!
E    Device 1: admin_closed was not restored from waku-backup!!
E    Device 1: member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES6Y1E7DObvdA3T0d3">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (9)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781091">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESGdF4IeiBodfXURU6">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESiihhMtgt8qHskGaz">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781097">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781099">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781096">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781098">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781095">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781094">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781093">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3781090">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuNukdtOcmo1izipK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES12sgN2PlPqLk9YHN">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/drawers/documentation_drawers/style.cljs
+++ b/src/quo/components/drawers/documentation_drawers/style.cljs
@@ -13,11 +13,11 @@
 
 (def content
   {:align-items   :flex-start
-   :margin-top    8
    :margin-bottom (+ (safe-area/get-bottom) 12)})
 
 (defn title
   [theme]
-  {:color (colors/theme-colors colors/neutral-100
-                               colors/white
-                               theme)})
+  {:color         (colors/theme-colors colors/neutral-100
+                                       colors/white
+                                       theme)
+   :margin-bottom 8})

--- a/src/quo/components/drawers/documentation_drawers/view.cljs
+++ b/src/quo/components/drawers/documentation_drawers/view.cljs
@@ -24,12 +24,13 @@
       :always-bounce-vertical            false
       :content-inset-adjustment-behavior :never}
      [rn/view {:style style/container}
-      [text/text
-       {:size                :heading-2
-        :accessibility-label :documentation-drawer-title
-        :style               (style/title theme)
-        :weight              :semi-bold}
-       title]
+      (when title
+        [text/text
+         {:size                :heading-2
+          :accessibility-label :documentation-drawer-title
+          :style               (style/title theme)
+          :weight              :semi-bold}
+         title])
       [rn/view {:style style/content :accessibility-label :documentation-drawer-content}
        content
        (when show-button?

--- a/src/status_im/contexts/keycard/backup/events.cljs
+++ b/src/status_im/contexts/keycard/backup/events.cljs
@@ -28,7 +28,8 @@
  (fn [_]
    {:fx [[:dispatch
           [:keycard/connect
-           {:on-error
+           {:theme :dark
+            :on-error
             (fn [error]
               (if (= error :keycard/error.keycard-blank)
                 (rf/dispatch [:keycard/backup.generate-and-load-key])
@@ -48,7 +49,8 @@
  (fn [_]
    {:fx [[:dispatch
           [:keycard/connect
-           {:on-error
+           {:theme :dark
+            :on-error
             (fn [error]
               (if (= error :keycard/error.keycard-blank)
                 (do
@@ -73,7 +75,8 @@
   [pin]
   (rf/dispatch
    [:keycard/connect
-    {:on-error
+    {:theme :dark
+     :on-error
      (fn [error]
        (if (= error :keycard/error.keycard-blank)
          (rf/dispatch
@@ -89,7 +92,8 @@
   [pin]
   (rf/dispatch
    [:keycard/connect
-    {:on-error
+    {:theme :dark
+     :on-error
      (fn [error]
        (if (= error :keycard/error.keycard-blank)
          (rf/dispatch

--- a/src/status_im/contexts/keycard/change_pin/events.cljs
+++ b/src/status_im/contexts/keycard/change_pin/events.cljs
@@ -68,5 +68,6 @@
          unmasked-new-pin              (security/safe-unmask-data new-pin)]
      {:fx [[:dispatch
             [:keycard/connect
-             {:key-uid    (get-in db [:profile/profile :key-uid])
+             {:theme      :dark
+              :key-uid    (get-in db [:profile/profile :key-uid])
               :on-success #(verify-pin-and-continue unmasked-current-pin unmasked-new-pin)}]]]})))

--- a/src/status_im/contexts/keycard/check/events.cljs
+++ b/src/status_im/contexts/keycard/check/events.cljs
@@ -7,6 +7,7 @@
      {:fx [[:dispatch
             [:keycard/connect
              {:key-uid (get-in db [:profile/profile :key-uid])
+              :theme :dark
               :on-success
               (fn []
                 (rf/dispatch [:keycard/disconnect])

--- a/src/status_im/contexts/keycard/check/view.cljs
+++ b/src/status_im/contexts/keycard/check/view.cljs
@@ -36,6 +36,7 @@
        :right-side [{:icon-name :i/info
                      :on-press  #(rf/dispatch [:show-bottom-sheet
                                                {:content keycard-documentation
+                                                :theme   :dark
                                                 :shell?  true}])}]}]
      [quo/page-top
       {:title            (i18n/label :t/check-keycard)

--- a/src/status_im/contexts/keycard/create/events.cljs
+++ b/src/status_im/contexts/keycard/create/events.cljs
@@ -7,7 +7,8 @@
  (fn [_]
    {:fx [[:dispatch
           [:keycard/connect
-           {:on-error
+           {:theme :dark
+            :on-error
             (fn [error]
               (if (= error :keycard/error.keycard-blank)
                 (do
@@ -18,8 +19,10 @@
 (defn- backup-recovery-phrase-success
   [masked-seed-phrase]
   (rf/dispatch [:navigate-back])
-  (rf/dispatch [:open-modal :screen/confirm-backup
+  (rf/dispatch [:open-modal :screen/confirm-backup-dark
                 {:masked-seed-phrase masked-seed-phrase
+                 :theme              :dark
+                 :shell?             true
                  :on-try-again       #(rf/dispatch [:open-modal :screen/backup-recovery-phrase-dark
                                                     {:on-success         backup-recovery-phrase-success
                                                      :masked-seed-phrase (->> masked-seed-phrase
@@ -81,7 +84,8 @@
  (fn [_]
    {:fx [[:dispatch
           [:keycard/connect
-           {:on-error #(rf/dispatch [:keycard/create.on-application-info-error %])}]]]}))
+           {:theme    :dark
+            :on-error #(rf/dispatch [:keycard/create.on-application-info-error %])}]]]}))
 
 (defn get-application-info-and-continue
   []

--- a/src/status_im/contexts/keycard/create/view.cljs
+++ b/src/status_im/contexts/keycard/create/view.cljs
@@ -2,9 +2,7 @@
   (:require [quo.core :as quo]
             [react-native.core :as rn]
             [status-im.common.events-helper :as events-helper]
-            [status-im.common.not-implemented :as not-implemented]
             [status-im.common.resources :as resources]
-            [status-im.config :as config]
             [status-im.constants :as constants]
             [status-im.contexts.keycard.common.view :as common.view]
             [utils.i18n :as i18n]
@@ -40,15 +38,24 @@
    [rn/view {:style {:flex 1}}]
    [common.view/tips]])
 
+(defn- ready-to-add-documentation
+  []
+  [quo/documentation-drawers
+   {:shell? true}
+   [quo/text {:size :paragraph-2}
+    (i18n/label :t/ready-add-keypair-keycard-documentation)]])
+
 (defn ready-to-add
   []
   [:<>
    [quo/page-nav
     {:icon-name  :i/close
      :on-press   events-helper/navigate-back
-     :right-side [(when config/show-not-implemented-features?
-                    {:icon-name :i/info
-                     :on-press  not-implemented/alert})]}]
+     :right-side [{:icon-name :i/info
+                   :on-press  #(rf/dispatch [:show-bottom-sheet
+                                             {:content ready-to-add-documentation
+                                              :theme   :dark
+                                              :shell?  true}])}]}]
    [quo/page-top
     {:title            (i18n/label :t/ready-add-keypair-keycard)
      :description      :text

--- a/src/status_im/contexts/keycard/error/view.cljs
+++ b/src/status_im/contexts/keycard/error/view.cljs
@@ -60,4 +60,6 @@
           :description-props {:text (i18n/label :t/remove-keycard-content)}
           :on-press          (fn []
                                (rf/dispatch [:show-bottom-sheet
-                                             {:content factory-reset/sheet}]))}])]]))
+                                             {:theme   :dark
+                                              :shell?  true
+                                              :content factory-reset/sheet}]))}])]]))

--- a/src/status_im/contexts/keycard/events.cljs
+++ b/src/status_im/contexts/keycard/events.cljs
@@ -90,7 +90,8 @@
  (fn [_ [{:keys [pin derivation-path key-uid account-preferences]}]]
    {:fx [[:dispatch
           [:keycard/connect
-           {:key-uid key-uid
+           {:theme :dark
+            :key-uid key-uid
             :on-success
             (fn []
               (rf/dispatch
@@ -160,7 +161,7 @@
                                      :keycard/error.not-keycard]))))}}))
 
 (rf/reg-event-fx :keycard/connect
- (fn [{:keys [db]} [{:keys [key-uid on-success on-error on-connect-event-vector]}]]
+ (fn [{:keys [db]} [{:keys [key-uid on-success on-error on-connect-event-vector theme]}]]
    (let [event-vector
          (or on-connect-event-vector
              [:keycard/get-application-info
@@ -170,6 +171,7 @@
      {:db (assoc-in db [:keycard :on-card-connected-event-vector] event-vector)
       :fx [[:dispatch
             [:keycard/show-connection-sheet
-             {:on-cancel-event-vector [:keycard/cancel-connection]}]]
+             {:on-cancel-event-vector [:keycard/cancel-connection]
+              :theme                  theme}]]
            (when (get-in db [:keycard :card-connected?])
              [:dispatch event-vector])]})))

--- a/src/status_im/contexts/keycard/factory_reset/view.cljs
+++ b/src/status_im/contexts/keycard/factory_reset/view.cljs
@@ -22,7 +22,8 @@
   [key-uid]
   (rf/dispatch
    [:keycard/connect
-    {:key-uid    key-uid
+    {:theme      :dark
+     :key-uid    key-uid
      :on-success reset-card
      :on-error   (fn [error]
                    (if (or (= error :keycard/error.keycard-frozen)

--- a/src/status_im/contexts/keycard/login/events.cljs
+++ b/src/status_im/contexts/keycard/login/events.cljs
@@ -43,7 +43,8 @@
  (fn []
    {:fx [[:dispatch
           [:keycard/connect
-           {:on-error
+           {:theme :dark
+            :on-error
             (fn [error]
               (if (= error :keycard/error.keycard-wrong-profile)
                 (do
@@ -57,7 +58,8 @@
  (fn [{:keys [db]} [pin]]
    {:fx [[:dispatch
           [:keycard/connect
-           {:key-uid    (get-in db [:keycard :application-info :key-uid])
+           {:theme      :dark
+            :key-uid    (get-in db [:keycard :application-info :key-uid])
             :on-success (fn []
                           (rf/dispatch
                            [:keycard/get-more-keys

--- a/src/status_im/contexts/keycard/manage/profile_keys/view.cljs
+++ b/src/status_im/contexts/keycard/manage/profile_keys/view.cljs
@@ -40,7 +40,9 @@
         :description-props {:text (i18n/label :t/change-pin-keycard-description)}
         :on-press          (fn []
                              (rf/dispatch [:show-bottom-sheet
-                                           {:content change-pin.view/change-pin-confirmation-sheet}]))}]
+                                           {:theme   :dark
+                                            :shell?  true
+                                            :content change-pin.view/change-pin-confirmation-sheet}]))}]
       [quo/settings-item
        {:title             (i18n/label :t/backup-keycard)
         :image             :icon
@@ -50,4 +52,7 @@
         :description       :text
         :description-props {:text (i18n/label :t/create-backup-profile-keycard)}
         :on-press          (fn []
-                             (rf/dispatch [:show-bottom-sheet {:content backup-sheet}]))}]]]))
+                             (rf/dispatch [:show-bottom-sheet
+                                           {:theme   :dark
+                                            :shell?  true
+                                            :content backup-sheet}]))}]]]))

--- a/src/status_im/contexts/keycard/migrate/events.cljs
+++ b/src/status_im/contexts/keycard/migrate/events.cljs
@@ -46,6 +46,7 @@
           [:keycard/connect
            {:key-uid (get-in db [:profile/profile :key-uid])
             :on-success #(rf/dispatch [:keycard/migration.continue])
+            :theme :dark
             :on-error
             (fn [error]
               (if (= error :keycard/error.keycard-blank)
@@ -55,8 +56,10 @@
 (defn- backup-recovery-phrase-success
   [masked-seed-phrase]
   (rf/dispatch [:navigate-back])
-  (rf/dispatch [:open-modal :screen/confirm-backup
+  (rf/dispatch [:open-modal :screen/confirm-backup-dark
                 {:masked-seed-phrase masked-seed-phrase
+                 :theme              :dark
+                 :shell?             true
                  :on-try-again       #(rf/dispatch [:open-modal :screen/backup-recovery-phrase-dark
                                                     {:on-success         backup-recovery-phrase-success
                                                      :masked-seed-phrase (->> masked-seed-phrase

--- a/src/status_im/contexts/keycard/nfc/sheets/events.cljs
+++ b/src/status_im/contexts/keycard/nfc/sheets/events.cljs
@@ -3,11 +3,12 @@
             [react-native.platform :as platform]))
 
 (rf/reg-event-fx :keycard/show-connection-sheet
- (fn [{:keys [db]} [{:keys [on-cancel-event-vector]} :as args]]
+ (fn [{:keys [db]} [{:keys [on-cancel-event-vector theme]} :as args]]
    (if platform/android?
      {:db (assoc-in db
            [:keycard :connection-sheet-opts]
-           {:on-close #(rf/dispatch on-cancel-event-vector)})
+           {:on-close #(rf/dispatch on-cancel-event-vector)
+            :theme    theme})
       :fx [[:effects.keycard.ios/start-nfc nil]
            [:dismiss-keyboard true]
            [:show-nfc-sheet nil]]}

--- a/src/status_im/contexts/keycard/nfc/sheets/view.cljs
+++ b/src/status_im/contexts/keycard/nfc/sheets/view.cljs
@@ -8,9 +8,9 @@
 
 (defn connect-keycard
   []
-  (let [connected?         (rf/sub [:keycard/connected?])
-        {:keys [on-close]} (rf/sub [:keycard/connection-sheet-opts])
-        theme              (quo.theme/use-theme)]
+  (let [connected?               (rf/sub [:keycard/connected?])
+        {:keys [on-close theme]} (rf/sub [:keycard/connection-sheet-opts])
+        theme                    (or theme (quo.theme/use-theme))]
     [rn/view {:flex 1}
      [rn/view {:flex 1}]
      [rn/view

--- a/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
+++ b/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
@@ -92,15 +92,16 @@
 
 (defn view
   []
-  (let [random-indices               (random-selection)
-        quiz-index                   (reagent/atom 0)
-        incorrect-count              (reagent/atom 0)
-        show-error?                  (reagent/atom false)
+  (let [random-indices         (random-selection)
+        quiz-index             (reagent/atom 0)
+        incorrect-count        (reagent/atom 0)
+        show-error?            (reagent/atom false)
         {:keys [on-success
                 on-try-again
-                masked-seed-phrase]} (rf/sub [:get-screen-params])
-        unmasked-seed-phrase         (security/safe-unmask-data masked-seed-phrase)
-        random-phrase                (reagent/atom [])]
+                masked-seed-phrase
+                theme shell?]} (rf/sub [:get-screen-params])
+        unmasked-seed-phrase   (security/safe-unmask-data masked-seed-phrase)
+        random-phrase          (reagent/atom [])]
     (fn []
       (rn/use-mount
        (fn []
@@ -119,12 +120,16 @@
                                                   (reset! show-error? false)
                                                   (when (and on-success (= @quiz-index questions-count))
                                                     (rf/dispatch [:show-bottom-sheet
-                                                                  {:content (fn [] [complete-backup-sheet
+                                                                  {:theme   theme
+                                                                   :shell?  shell?
+                                                                   :content (fn [] [complete-backup-sheet
                                                                                     on-success])}])))
                                                 (do
                                                   (when (> @incorrect-count 0)
                                                     (rf/dispatch [:show-bottom-sheet
-                                                                  {:content (fn []
+                                                                  {:theme   theme
+                                                                   :shell?  shell?
+                                                                   :content (fn []
                                                                               [cheat-warning
                                                                                on-try-again])}]))
                                                   (reset! incorrect-count (inc @incorrect-count))

--- a/src/status_im/navigation/screens.cljs
+++ b/src/status_im/navigation/screens.cljs
@@ -925,6 +925,7 @@
    {:name      :screen/keycard.empty-create
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.empty/create}
 
@@ -1024,30 +1025,35 @@
    {:name      :screen/keycard.create-profile
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.create/view}
 
    {:name      :screen/keycard.create.ready-to-add
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.create/ready-to-add}
 
    {:name      :screen/keycard.ready-to-change-pin
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.change-pin/ready-to-change-pin}
 
    {:name      :screen/keycard.pin-change-success
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.change-pin/pin-change-success}
 
    {:name      :screen/keycard.pin-change-failed
     :metrics   {:track? true}
     :options   {:insets                 {:top? true :bottom? true}
+                :theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component keycard.change-pin/pin-change-failed}
 
@@ -1168,6 +1174,12 @@
      :options   {:theme  :dark
                  :insets {:top? true :bottom? true}}
      :component backup-recovery-phrase/view}
+
+    {:name      :screen/confirm-backup-dark
+     :metrics   {:track? true}
+     :options   {:insets {:top? true :bottom? true}
+                 :theme  :dark}
+     :component wallet-confirm-backup/view}
 
     {:name      :screen/use-recovery-phrase
      :metrics   {:track? true}

--- a/translations/en.json
+++ b/translations/en.json
@@ -2131,6 +2131,7 @@
   "read": "Read",
   "read-more": "Read more",
   "ready-add-keypair-keycard": "Ready to add key pair to Keycard",
+  "ready-add-keypair-keycard-documentation": "When you create a Status profile, the Status app generates cryptographic keys for your profile authentication and authorization.\n\nYour Keycard stores the cryptographic keys for your Status profile, permanently offline and isolated from the internet, and also secures your Wallet accounts.",
   "ready-keycard": "Get your Keycard ready",
   "ready-to-change-pin": "Ready to change PIN",
   "ready-to-migrate-key-pair": "Ready to migrate profile key pair to the Keycard",

